Chatkit/Redux/Store/State.swift

```diff
@@ -0,0 +1,34 @@
+
+enum Internal {}
+
```

Comment:
Where are unit tests that test `State`, `Room` and `User` in isolation?

Reply:
All these classes are just stubbed (not a test sense) for you to flesh out.  They are effectively not implemented yet.  You will need to write these tests as part of your work on adding the  `State` and `Reducer`s.